src/main/java/com/answerdigital/answerking/exception/util/ValidationErrorResponse.java

```diff
@@ -0,0 +1,44 @@
+package com.answerdigital.answerking.exception.util;
+
+import com.answerdigital.answerking.exception.custom.AnswerKingValidationException;
+import lombok.Getter;
+import lombok.extern.slf4j.Slf4j;
+
+import javax.servlet.http.HttpServletRequest;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.Map;
+
+@Slf4j
+@Getter
+public class ValidationErrorResponse {
```

Comment:
I tried that, but if I do then other errors apart of Validation still prints empty error map which is against openApi contract, so that's why I had to create a new one.